applications/NeuralNetworkApplication/python_scripts/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Application dependent names and paths
```

Comment:
I am quite sure this is not required, other applications are not defining this: https://github.com/KratosMultiphysics/Kratos/tree/master/applications/StructuralMechanicsApplication/python_scripts